src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Components/Identity/ExternalLoginPicker.razor

```diff
@@ -0,0 +1,47 @@
+﻿@using Microsoft.AspNetCore.Authentication
+@using Microsoft.AspNetCore.Identity
+@using BlazorWeb_CSharp.Components.Pages.Account
+@using BlazorWeb_CSharp.Data
+@using BlazorWeb_CSharp.Identity
+
+@inject SignInManager<ApplicationUser> SignInManager
+@inject IdentityRedirectManager RedirectManager
+
+@if ((_externalLogins?.Count ?? 0) == 0)
+{
+    <div>
+        <p>
+            There are no external authentication services configured. See this <a href="https://go.microsoft.com/fwlink/?LinkID=532715">article
+            about setting up this ASP.NET application to support logging in via external services</a>.
+        </p>
+    </div>
+}
+else
+{
+    <form id="external-account" class="form-horizontal" action="/Account/PerformExternalLogin" method="post">
```

Comment:
```suggestion
    <form id="external-account" class="form-horizontal" action="Account/PerformExternalLogin" method="post">
```

We don't use root-relative URLs as they aren't compatible with `<base href>`.

Reply:
Also is the `id` required for some reason? Is this used for styling? We should prefer `class` for styling, and not add an `id` if it's not required.